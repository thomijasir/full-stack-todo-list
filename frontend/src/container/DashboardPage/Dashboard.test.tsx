import React from 'react';
import renderer from 'react-test-renderer';
import Dashboard from './Dashboard.page';

test('Render Dashboard Page', () => {
  const element = renderer.create(<Dashboard />).toJSON();
  expect(element).toMatchSnapshot();
});
