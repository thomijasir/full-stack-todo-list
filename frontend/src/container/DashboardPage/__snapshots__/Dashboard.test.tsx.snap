// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Render Dashboard Page 1`] = `
<div>
  <h1>
    Its Dashboard Screen
  </h1>
</div>
`;
